move_robot_pose_node:
  ros__parameters:
    positions:
      correct_rotation_deg: 90.0

      line_targets:
        - 1.249
        - 1.231
        - 1.230
        - 1.650
        - 1.762
        - 1.790

      poses:
        0:
          home:       [-90.31, 11.78, 0.21, 0.28, 76.70, -20.20]
          pre_pick:   [-15.50, 53.11, -16.97, 0.00, 53.85, -195.50]
          pick:       [-15.50, 62.44, -13.69, 0.00, 41.25, -195.50]
          pre_place:  [-143.41, 29.99, 34.40, 0.00, 25.60, 36.59]
          place:      [-143.41, 45.69, 36.96, 0.00, 7.36, 36.59]

        1:
          home:       [-90.31, 11.78, 0.21, 0.28, 76.70, -20.20]
          pre_pick:   [-16.60, 47.08, -4.15, 0.00, 47.07, -196.66]
          pick:       [-16.66, 57.38, -1.14, 0.00, 33.75, -196.66]
          pre_place:  [-138.76, 26.16, 42.28, 0.00, 21.55, 41.24]
          place:      [-138.76, 43.83, 45.03, 0.00, 1.14, 41.24]

        2:
          home:       [-90.31, 11.78, 0.21, 0.28, 76.70, -20.20]
          pre_pick:   [-18.34, 40.71, 9.19, 0.00, 40.09, -198.34]
          pick:       [-18.34, 52.30, 12.11, 0.00, 25.59, -198.34]
          pre_place:  [-134.23, 23.34, 48.02, 0.00, 18.64, 45.77]
          place:      [-134.23, 42.83, 50.97, 0.00, -3.80, 45.77]

        3:
          home:       [-90.31, 11.78, 0.21, 0.28, 76.70, -20.20]
          pre_pick:   [-37.43, 37.23, 16.39, 0.00, 36.38, 142.57]
          pick:       [-37.43, 49.69, 19.33, 0.00, 20.98, 142.57]
          pre_place:  [-135.29, 26.85, 40.87, 0.00, 22.28, 44.71]
          place:      [-135.29, 44.12, 43.57, 0.00, 2.30, 44.71]

        4:
          home:       [-90.31, 11.78, 0.21, 0.28, 76.70, -20.20]
          pre_pick:   [-48.68, 27.28, 36.45, 0.00, 26.27, 131.32]
          pick:       [-48.68, 43.27, 39.71, 0.00, 7.02, 131.32]
          pre_place:  [-135.97, 27.30, 39.95, 0.00, 22.74, 44.03]
          place:      [-135.97, 44.32, 42.64, 0.00, 3.04, 44.03]

        5:
          home:       [-90.31, 11.78, 0.21, 0.28, 76.70, -20.20]
          pre_pick:   [-58.88, 22.35, 45.98, 0.00, 21.66, 121.12]
          pick:       [-58.88, 41.02, 49.60, 0.00, -0.62, 121.12]
          pre_place:  [-131.63, 24.73, 45.20, 0.00, 20.06, 48.37]
          place:      [-131.63, 43.27, 48.05, 0.00, -1.32, 48.37]
